packages/flutter/lib/src/material/material_button.dart

```diff
@@ -415,6 +415,7 @@ class MaterialButton extends StatelessWidget {
       animationDuration: buttonTheme.getAnimationDuration(this),
       child: child,
       materialTapTargetSize: materialTapTargetSize ?? theme.materialTapTargetSize,
+      disabledElevation: disabledElevation ?? 0.0,
```

Comment:
Is there already a test for the default value? Just to make sure that we don't regress to not using `0.0`.

Reply:
Actually no, but let me add one right now :)